package.json

```diff
@@ -24,6 +24,7 @@
     "json2csv": "3.7.0",
     "moment": "2.24.0",
     "moment-timezone": "0.5.10",
+    "node-schedule": "^1.3.2",
```

Comment:
I'd appreciate it if you could provide the corresponding yarn.lock file.

Reply:
hmm, i got massive issue getting this website to run on windows. When I fix some issue the next one pops up...

Reply:
Maybe I should install me Linux again I remeber it was easy there